src/database/query/youfocus.ts

```diff
@@ -150,89 +133,79 @@ const deleteUserPlaylistFromDB = async (
   }
 };
 
-const updateUserPlaylistWatchTime = async (
+const updateUserPlaylistData = async (
```

Comment:
Rewrite this function with simple steps - 

1. Check if User Playlist exists
2. If Yes, Update Recommended or Timer


Don't overcomplicate for no reason.